<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassifyCategoriesWindow</class>
 <widget class="QMainWindow" name="ClassifyCategoriesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="MainFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1107</width>
      <height>756</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2b3e47;
border-radius: 25px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="AnimalsButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #2b3e47, stop:1 #f16150);
	color:#fff;
	border-radius:10px;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(63, 102, 120), stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font-size : 25px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../HomeScreen/home_screen.qrc">
       <normaloff>:/Images/Images/Classify Icons/Animals.png</normaloff>:/Images/Images/Classify Icons/Animals.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QFrame" name="circle1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>351</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/Images/circle1.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="logo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/Images/logoAppWhite.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="circle2">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>30</y>
       <width>631</width>
       <height>761</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/Images/circle1.png);
background-color: transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="GoBack">
      <property name="geometry">
       <rect>
        <x>254</x>
        <y>670</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
image:none;
color: #fff;
font: 11pt &quot;Futura Md BT&quot;;
</string>
      </property>
      <property name="text">
       <string>Go Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="circle3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>190</y>
       <width>211</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/Images/circle2.png);
background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="circle4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>360</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/Images/circle2.png);
background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="windowNav">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>10</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="minimizeButton">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../HomeScreen/home_screen.qrc">
          <normaloff>:/Images/Images/minimize.png</normaloff>:/Images/Images/minimize.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../HomeScreen/home_screen.qrc">
          <normaloff>:/Images/Images/close.png</normaloff>:/Images/Images/close.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="main_header">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1111</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="robot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>680</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/Images/robot.png);
background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="PlantsButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>110</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #2b3e47, stop:1 #f16150);
	color:#fff;
	border-radius:10px;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(63, 102, 120), stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font-size : 25px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../HomeScreen/home_screen.qrc">
       <normaloff>:/Images/Images/Classify Icons/Plants.png</normaloff>:/Images/Images/Classify Icons/Plants.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="VehicleButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>110</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #2b3e47, stop:1 #f16150);
	color:#fff;
	border-radius:10px;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(63, 102, 120), stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font-size : 25px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../HomeScreen/home_screen.qrc">
       <normaloff>:/Images/Images/Classify Icons/vehicles.png</normaloff>:/Images/Images/Classify Icons/vehicles.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="LogosButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>310</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #2b3e47, stop:1 #f16150);
	color:#fff;
	border-radius:10px;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(63, 102, 120), stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font-size : 25px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../HomeScreen/home_screen.qrc">
       <normaloff>:/Images/Images/Classify Icons/Logos.png</normaloff>:/Images/Images/Classify Icons/Logos.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="ArtistsButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>310</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #2b3e47, stop:1 #f16150);
	color:#fff;
	border-radius:10px;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(63, 102, 120), stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font-size : 25px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../HomeScreen/home_screen.qrc">
       <normaloff>:/Images/Images/Classify Icons/Artists.png</normaloff>:/Images/Images/Classify Icons/Artists.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="AthletesButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>310</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #2b3e47, stop:1 #f16150);
	color:#fff;
	border-radius:10px;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(63, 102, 120), stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font-size : 25px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../HomeScreen/home_screen.qrc">
       <normaloff>:/Images/Images/Classify Icons/Athletes.png</normaloff>:/Images/Images/Classify Icons/Athletes.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="MIButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>510</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #2b3e47, stop:1 #f16150);
	color:#fff;
	border-radius:10px;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(63, 102, 120), stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font-size : 25px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../HomeScreen/home_screen.qrc">
       <normaloff>:/Images/Images/Classify Icons/MusicInstruments.png</normaloff>:/Images/Images/Classify Icons/MusicInstruments.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="DishesButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>510</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #2b3e47, stop:1 #f16150);
	color:#fff;
	border-radius:10px;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(63, 102, 120), stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font-size : 25px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../HomeScreen/home_screen.qrc">
       <normaloff>:/Images/Images/Classify Icons/Dishes.png</normaloff>:/Images/Images/Classify Icons/Dishes.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="ClothesButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>510</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #2b3e47, stop:1 #f16150);
	color:#fff;
	border-radius:10px;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(63, 102, 120), stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font-size : 25px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../HomeScreen/home_screen.qrc">
       <normaloff>:/Images/Images/Classify Icons/Clothes.png</normaloff>:/Images/Images/Classify Icons/Clothes.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
    <zorder>logo</zorder>
    <zorder>circle1</zorder>
    <zorder>circle2</zorder>
    <zorder>circle3</zorder>
    <zorder>AnimalsButton</zorder>
    <zorder>circle4</zorder>
    <zorder>main_header</zorder>
    <zorder>windowNav</zorder>
    <zorder>robot</zorder>
    <zorder>PlantsButton</zorder>
    <zorder>VehicleButton</zorder>
    <zorder>LogosButton</zorder>
    <zorder>ArtistsButton</zorder>
    <zorder>AthletesButton</zorder>
    <zorder>MIButton</zorder>
    <zorder>DishesButton</zorder>
    <zorder>ClothesButton</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../HomeScreen/home_screen.qrc"/>
 </resources>
 <connections/>
</ui>
